tensorizer/stream_io.py

```diff
@@ -146,6 +156,8 @@ def __init__(
             curl_path,
             "--header",
             "Accept-Encoding: identity",
+            "--dump-header",
+            "/dev/stderr",
```

Comment:
nit: re-using stderr for headers is slightly problematic because you might not get the errors clearly, or they might be mixed with the headers. Fix later, use a seperate FD or tempfile for the header

Reply:
Added a `TODO`.

Reply:
Fixed this by using a separate pipe in 558176610d5b159b7bb7cb97304992f99e511b4f and writing to it through `/dev/fd`